content/en/references/configuration.md

```diff
@@ -308,40 +312,74 @@ To learn more about these configuration options, see [DNS Server]({{< ref "dns-s
-| `S3_DIR` | | **Deprecated.** This is only supported for the `legacy_v2` provider. Configure a global parent directory that contains all buckets as sub-directories (`S3_DIR=/path/to/root`) or an individual directory that will get mounted as special bucket names (`S3_DIR=/path/to/root/bucket1:bucket1`). Only available for Localstack Pro.
-
 ## Legacy
 
 These configurations have already been removed and **won't have any effect** on newer versions of LocalStack.
 **Please remove them from your configuration.**
 
-| Variable | Removed in | Description |
-| - | - | - |
-| `DATA_DIR`| 2.0.0 | **Legacy.** Local directory for saving persistent data. Use `PERSISTENCE` instead. |
-| `HOST_TMP_FOLDER` | 2.0.0 | **Legacy.** Temporary folder on the host that gets mounted as `$TMPDIR/localstack` into the LocalStack container. Required only for Lambda volume mounts when using `LAMBDA_REMOTE_DOCKER=false.` |
-| `INIT_SCRIPTS_PATH` | 2.0.0 | **Legacy.** Before 1.0, this was used to configure the path to the initializing files with extensions `.sh` that were found in `/docker-entrypoint-initaws.d`. This has been replaced by the [init-hook system](https://docs.localstack.cloud/references/init-hooks/). |
-| `TMPDIR`| 2.0.0 | **Legacy.** Temporary folder on the host running the CLI and inside the LocalStack container .|
-| `<SERVICE>_BACKEND` | 3.0.0 | **Legacy.** Custom endpoint URL to use for a specific service, where `<SERVICE>` is the uppercase service name.
-| `ACTIVATE_NEW_POD_CLIENT` | 3.0.0 | **Legacy.** Whether to use the new Cloud Pods client leveraging LocalStack container's APIs. |
-| `ES_CUSTOM_BACKEND` | 3.0.0 | **Legacy.** Use [`OPENSEARCH_CUSTOM_BACKEND`](#opensearch) instead. URL to a custom elasticsearch backend cluster. If this is set to a valid URL, then localstack will not create elasticsearch cluster instances, but instead forward all domains to the given backend (see [Custom Elasticsearch Backends]({{< ref "elasticsearch#custom-elasticsearch-backends" >}})). |
-| `ES_ENDPOINT_STRATEGY` | 3.0.0 | **Legacy.** Use [`OPENSEARCH_ENDPOINT_STRATEGY`](#opensearch) instead. Governs how domain endpoints are created to access a cluster (see [Elasticsearch Endpoints]({{< ref "elasticsearch#endpoints" >}})) |
-| `ES_MULTI_CLUSTER` | 3.0.0 | **Legacy.** Use [`OPENSEARCH_MULTI_CLUSTER`](#opensearch) instead. When activated, LocalStack will spawn one Elasticsearch cluster per domain. Otherwise all domains will share a single cluster instance. This is ignored if `ES_CUSTOM_BACKEND` is set. |
-| `MOCK_UNIMPLEMENTED` | 3.0.0 | **Legacy.** Whether to return mocked success responses (instead of 501 errors) for currently unimplemented API methods
+| Variable | Removed in | Example Values | Description |
+| - | - | - | - |
+| `PORT_WEB_UI` | 0.12.8 | |  |
```

Comment:
I don't like to scroll down over pages of super old legacy configuration. It would be nicer to see the most recent ones at the top by inverting the order 🥇

Reply:
good point and I agree :+1: